contracts/OpenStream.sol

```diff
@@ -66,4 +87,10 @@ contract OpenStream is ReentrancyGuard, IOpenStream {
         emit TokensClaimed(claimableAmount);
     }
 
+    function terminate() external {
```

Comment:
@accompany1205 Like everyone can call this function, is that correct?

Reply:
@georgeciubotaru Only payees can call this function.
When creating open streams, payees address is stored

Reply:
@accompany1205 I cannot see that, it's external and without checks, that means everyone can call it

Reply:
@georgeciubotaru You can check line 73. I used modifier.
You can also find onlyPayee modifier in line47

Reply:
I added a modifier, onlyPayer to the terminate function.
Could you check it again?